aten/src/ATen/native/LinearAlgebra.cpp

```diff
@@ -350,13 +360,17 @@ Tensor &addmm_cpu_(Tensor& self, const Tensor& mat1, const Tensor& mat2, Scalar
 }
 
 Tensor& mm_cpu_out(Tensor & result, const Tensor & self, const Tensor & mat2) {
-  result.resize_({ self.size(0), mat2.size(1) });
+  TORCH_CHECK(self.dim() == 2, "self must be a matrix");
+  TORCH_CHECK(mat2.dim() == 2, "mat2 must be a matrix");
+  native::resize_(result, {self.sizes()[0], mat2.sizes()[1]});
```

Comment:
nit: since you are resizing result in addmm_cpu_impl, do you need this resize here?

Reply:
`result` is also the `self` tensor in `addmm` which must at least broadcast to the correct dimensions.